File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.js
function solution(ineq, eq, n, m) {
    var answer = true;
    
    if (ineq === '>' && eq === '=') {
        answer = n >= m
    } else if (ineq === '>' && eq === '!') {
        answer = n > m
    } else if (ineq === '<' && eq === '=') {
        answer = n <= m
    } else {
        answer = n < m
    }

    if (answer === true) return 1
    else return 0
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    answer = True
    
    if ineq == '>' and eq == '=':
        answer = n >= m
    elif ineq == '>' and eq == '!':
        answer = n > m
    elif ineq == '<' and eq == '=':
        answer = n <= m
    else:
        answer = n < m
        
    return 1 if answer else 0